components/Map.js

```diff
@@ -51,6 +65,14 @@ export default function Map() {
           );
         })}
       </StyledMapContainer>
+      <StyledPositionBtn onClick={handleSetView}>
```

Comment:
in your prev this doesnt work for me :/

Reply:
I add an if else block, if the app does not be able to use your location, there will be an altert on your browser :)